views/layout.html

```diff
@@ -217,22 +217,7 @@
         </div>
 
         <div class="footer__social">
-          <p>
-          <a href="https://www.facebook.com/contentful/" target="_blank" rel="noopener">
-            <svg>
-              <use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#facebook"></use>
-            </svg>
-          </a>
-          <a href="https://twitter.com/contentful" target="_blank" rel="noopener">
-            <svg>
-              <use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#twitter"></use>
-            </svg>
-          </a>
-          <a href="https://github.com/contentful" target="_blank" rel="noopener">
-            <svg>
-              <use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#github"></use>
-            </svg>
-          </a>
+          <p><a href="https://www.facebook.com/contentful/" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#facebook"></use></svg></a><a href="https://twitter.com/contentful" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#twitter"></use></svg></a><a href="https://github.com/contentful" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#github"></use></svg></a>
```

Comment:
I think you don't need to remove the whitespace between 2 different `a` tags, just the one _inside_.
So you can still have 3 `a` tags on 3 different lines:
``` html
<p>
  <a>...</a>
  <a>...</a>
  <a>...</a>
</p>
```